bin/composer_install.sh

```diff
@@ -19,16 +19,12 @@ case "${dependency_versions}" in
     *) composer_command="install" ;;
 esac
 
-IFS=' '
-additional_options=("${additional_composer_options}")
-for option in "${additional_options[@]}"; do
-    composer_options+=("${option}")
-done
+read -r -a additional_options <<<"${additional_composer_options}"
```

Comment:
Oh, nice! I didn't think about using redirection with `read`. I was thinking of it only in terms of user interaction.